README.md

```diff
@@ -156,14 +158,14 @@ Here is a good editor for regexp, maybe try and do it without using it first tho
 File.dirname(__FILE__)
 ```
 
-- *give me answer*
+- precious
 
 ```ruby
 # /User/tom/junk/ring.rb
 File.path(__dir__)
 ```
 
-- *give me answer*
+- /User/tom/junk
```

Comment:
This was kind of a trick question because `File.path` is redundant here and will return the same as `__dir__`: `/User/tom/junk`

https://ruby-doc.org/core-2.5.1/Kernel.html#method-i-__dir__`